portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,35 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+    <header>
+        <nav class="menu">
+            <ul>
+                <li><a href ="#home">Home</a></li>
+                <li><a href ="#about-me">About Me</a></li>
+                <li><a href ="#projects">Projects</a></li>
+                <li><a href ="#contact-me">Contact Me</a></li>
+            </ul>
+        </nav>
+    </header>
+        <div class="content">
+            <div id="home">
```

Comment:
Alternatively, check out the `section` tag, which is a semantic way to represent different sections of an HTML page.